### PR TITLE
Disqus bug

### PR DESCRIPTION
**Pull request by somatheus**
The last disqus code it was not working. It was giving a message 'We were unable to load disqus', so I made some researches I could know that one variable was deprecated, so I'm uploading the new code :D

### CODE REVIEW

File: templates/article.html [hunk 1]
```diff
@@ -16,16 +16,6 @@
...
-    <meta property="og:type" content="article">
-    <meta property="og:title" content="{{ article.title|striptags }}">
-    <meta property="og:url" content="{{ article.url }}">
-    <meta property="og:description" content="{{ article.summary|striptags|escape }}">
-    <meta property="article:published_time" content="{{ article.date|strftime('%A, %B %d, %Y') }}">
-    <meta name="twitter:title" content="{{ article.title|striptags }}">
-    <meta name="twitter:description" content="{{ article.summary|striptags|escape }}">
-    <meta name="description" content="{{ article.summary|striptags|escape }}">
-    <meta name="twitter:image" content="{{ article.image }}">
-    <meta property="og:image" content="{{ article.image }}">
```
Comment (guilherme-toti) [inline templates/article.html:28]: Why did you remove these `metas`?

File: templates/article.html [hunk 2]
```diff
@@ -60,7 +50,7 @@
 	</figure>
 	{% endif %}
 
-	<section class="post-content">
```
Comment (guilherme-toti) [inline templates/article.html:63]: I think this shouldn't be removed too

File: templates/article.html [hunk 3]
```diff
@@ -94,16 +84,14 @@ <h2><a href="{{ SITEURL }}/{{ article.url }}">{{ article.title }}</a></h2>
...
-		            })();
-		        </script>
+		        <script>
+					(function() { // DON'T EDIT BELOW THIS LINE
+					var d = document, s = d.createElement('script');
+					s.src = 'disqus-user-name.disqus.com/embed.js';
+					s.setAttribute('data-timestamp', +new Date());
+					(d.head || d.body).appendChild(s);
+					})();
+				</script>
```
Comment (guilherme-toti) [inline templates/article.html:94]: The indentation here is not right

File: templates/disqus_script.html [hunk 4]
```diff
@@ -1,11 +1,9 @@
...
+<script>
+(	function() { // DON'T EDIT BELOW THIS LINE
+		var d = document, s = d.createElement('script');
+		s.src = 'disqus-user-name.disqus.com/embed.js';
+		s.setAttribute('data-timestamp', +new Date());
+		(d.head || d.body).appendChild(s);
+	})();
 </script>
-{% endif %}
+<noscript>Please enable JavaScript to view the</noscript>
```
Comment (guilherme-toti) [inline templates/disqus_script.html:9]: This is necessary? I think it's not being call in any place

**Review by guilherme-toti — CHANGES_REQUESTED** [review]

**Comment by coolapso** [comment]
no activity since the review. and all seems to be working perfectly. 
Closing.

**Pull request closed without merging**